src/react/workflow/WorkflowContent.jsx

```diff
@@ -23,6 +24,15 @@ type Props = {
 
 export const InfoWarning = ({ title }: { title: string}) => <Warning iconClass='fas fa-2x fa-info-circle' title={title} />;
 
+const ReplicationTitle = styled(Title)`
```

Comment:
These styled components can be moved to the `../ui-elements//TableKeyValue2` and be renamed `Title` and `Subtitle`. That would help UX consistency.

Reply:
done